src/operations/clean.rs

```diff
@@ -108,7 +109,7 @@ pub fn clean(options: Options) {
         }
 
         // Clear pacman's cache (keeping latest 3 versions of installed packages)
-        Command::new("sudo")
+        Command::new(config::read().bin.sudo.unwrap_or_default())
```

Comment:
Since bin itself is a `String` this would default to an empty string if no sudo binary was configured.

Reply:
It currently crashes if a config key isn't configured rn..

Reply:
Nice, I guess @not-my-segfault should have used the `config` crate *hint* *hint*

Reply:
yeah.. i feel like config should be static so we don't reread every time?

Reply:
That too, yes. My idea was to use lazy_static for that to only read it once